web/app/js/components/ServiceMesh.jsx

```diff
@@ -249,26 +253,37 @@ export default class ServiceMesh extends React.Component {
   }
 
   renderAddDeploymentsMessage() {
-    return (
-      <div className="mesh-message">
-        {this.unaddedDeploymentCount() === 0 ?
+    let instructions = (<div className="instructions">Add one or more deployments to the deployment.yml file <br /><br />
+                        Then run <code>conduit inject deployment.yml | kubectl apply -f - </code> to add deploys to the service mesh</div>);
```

Comment:
this indentation is a little weird. I'd prefer
```
    let instructions = (
      <div className="instructions">Add one or more deployments to the deployment.yml file <br /><br />
      Then run <code>conduit inject deployment.yml | kubectl apply -f - </code> to add deploys to the service mesh</div>
    );
```

Reply:
changed it 👍